coder/main.go

```diff
@@ -226,7 +277,7 @@ func Run(s *discordgo.Session, m *discordgo.MessageCreate) error {
 		s.ChannelMessageSend(m.ChannelID, ":robot: Failed to send reply. :( Maybe the output is too long?")
 	}
 
-	return nil
+	return nil	
```

Comment:
I'm sure gofmt or something else might pick up on this, but I don't want this tab on the end of a line!

Reply:
<3                 whitespace